lms/resources/_js_config.py

```diff
@@ -20,6 +20,7 @@ def __init__(self, context, request):
         self._consumer_key = (
             request.lti_user.oauth_consumer_key if request.lti_user else None
         )
+        self._authority = request.registry.settings["h_authority"]
```

Comment:
`JSConfig` was already using the authority setting (see below), I just added a convenience variable for it. `JSConfig` is one of the two places where it makes sense for the authority setting to be used: it's one of the "edges" of the app where we render userid's and groupid's for passing to the Hypothesis client, and those IDs must contain the authority